src/shared/components/instances.tsx

```diff
@@ -11,6 +11,22 @@ export class Instances extends Component<any, any> {
     super(props, context);
   }
 
+  render() {
+    return (
+      <div class="container">
+        <Helmet title={title}>
+          <meta property={"title"} content={title} />
+        </Helmet>
+        <h1 class="is-marginless">{i18n.t("lemmy_servers")}</h1>
+        {this.header()}
+        <br />
+        <br />
+        {this.render_list("Recommended", instance_stats.recommended)}
```

Comment:
I was hoping to randomize the recommended instances here, but that seems unnecessarily complicated in js/ts.